README.md

```diff
@@ -1,24 +1,42 @@
-# rbjks.github.io
-<a href="https://gitpod.io/#https://github.com/rbjks/rbjks.github.io" target="_blank">
-  <img src="https://gitpod.io/button/open-in-gitpod.svg" alt="Open RBJKS in Gitpod">
-</a>
+[![RBJKS NGO](assets/images/hero.png)](https://rbjks.github.io/index.html)
 
+[![RepoRater Badge](https://repo-rater.eddiehub.io/api/badge?owner=rbjks&name=rbjks.github.io)](https://repo-rater.eddiehub.io/rate?owner=rbjks&name=rbjks.github.io)
+[![Facebook Badge](https://img.shields.io/badge/Facebook-Profile-blue?style=flat&logo=facebook&link=https%3A%2F%2Fwww.facebook.com%2Fakil.bakhshi%2F)](https://www.facebook.com/akil.bakhshi/)
+[![Twitter Badge](https://img.shields.io/badge/Twitter-Profile-blue?style=flat&logo=twitter&link=https%3A%2F%2Ftwitter.com%2Fa_bsays)](https://twitter.com/a_bsays)
 
-Access this website at https://rbjks.github.io/index.html
 
-The aim for the website is as a static webpage for an NGO 'Ranjeet Bakhshi Jan Kalyan Sabha'. The NGO conducts many activites and crowd-funding campaigns to help people in lower-economic and social backgrounds to empower themselves.
-This website has to be duolingual with both English and Hindi as the required languages.
 
-Preferred colors for the website
 
-<span style="color:#22B9A8">Emerald - #22B9A8</span>
+# [RBJKS NGO](https://rbjks.github.io/index.html)
 
-![Emerald](https://user-images.githubusercontent.com/115361239/220578047-1ed3c681-0282-4b1e-85bd-66e86b1344bd.png)
+Welcome to the RBJKS (Ranjeet Bakhshi Jan Kalyan Sabha) NGO Project! Our mission is to address local community issues in Nagpur, Himachal Pradesh, India, through organic crowdfunding initiatives.
 
-<span style="color:#ff9933">Deep Saffron - #ff9933</span>
+## Table of Contents
 
-![Deep Saffron](https://user-images.githubusercontent.com/115361239/220579735-fb87b37d-67f4-406c-8188-2d42f29f9847.png)
```

Comment:
Please don't remove the color schemes as these are important to contributors. 
Perhaps we can add them as another section just below the Contact section?